charts/service-deployment/values.yaml

```diff
@@ -104,12 +104,12 @@ hpa:
   maxReplicas: 20
   # -- Default average CPU utilization before auto-scaling starts
   metrics:
-  - type: Resource
-    resource:
-      name: cpu
-      target:
-        type: Utilization
-        averageUtilization: 75
+    - type: Resource
```

Comment:
YAML indentation changes can cause issues - has a linter added this, as it looked to be correct before (i.e. dash directly under the start of metrics.